### PR TITLE
PWGGA/GammaConvBase  adding feature to look deeper in the decay chain of the photons to find additional true meson candidates, + plot to see differences to old previous version and NCell cut min energy bugfix

### PR DESCRIPTION
**Pull request by AMechler**

### CODE REVIEW

**Comment by dmuhlhei** [comment]
+1

**Comment by AMechler** [comment]
I am afraid I pushed some new things to quickly before the original PR was done

File: PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx [hunk 1]
```diff
@@ -2953,6 +2967,8 @@ void AliAnalysisTaskGammaConvCalo::UserCreateOutputObjects(){
         if (fIsMC > 1){
           fHistoTruePi0InvMassPt[iCut]->Sumw2();
           fHistoTrueEtaInvMassPt[iCut]->Sumw2();
+          fHistoTruePi0InvMassPtAdditional[iCut]->Sumw2();
+          fHistoTrueEtaInvMassPtAdditional[iCut]->Sumw2();
```
Comment (nschmidtALICE) [inline PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx:2971]: this needs to be in the "if (fDoLightOutput!=2 ){" condition

**Review by nschmidtALICE — CHANGES_REQUESTED** [review]

File: PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx (hunk 1 above)
Comment (AMechler) [reply PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx]: Yes you are correct, thanks for noticing.
I moved the lines accordingly.

**Comment by nschmidtALICE** [comment]
+1

**Pull request merged**